### ISSUE TITLE
BINDetect running indefinitely

### ISSUE THREAD

**Issue by adamxmiranda** [loosolab/TOBIAS#140]
I've been experiencing an issue while running comparative BINDetect on two different conditions. The progress of the function seems to be stalling out with the last output being:

`Calculating background log2 fold-changes between conditions
[INFO]	- E545K / H1047R

[INFO]Processing scanned TFBS individually
....
Progress 99.73%`

I'm not sure what the issue could be and I have tried remaking the input files thinking that there could be some kind of issue with them. Any help or insight to what the issue could be is appreciated.

My input is:
`TOBIAS BINDetect --motifs ${JASPAR} \
--signals ${PROCESSED_DIR}/mut1_footprints.bw ${PROCESSED_DIR}/mut2_footprints.bw \
--genome ${GENOME} \
--peaks ${NPEAK_DIR1}/intersected.bed \
--outdir ${PROCESSED_DIR}/motifs_sharedMerged_compared \
--cond_names mut1 mut2 \
--cores 12`

**Comment by msbentsen** [comment]
Hi, 

Thank you for the issue - I am not 100% sure what the problem is. Could you try to run the test-data for TOBIAS? I would be interested to see if it is a system or a data issue. You can obtain the test data as described [here](https://github.com/loosolab/TOBIAS/wiki/test-data), and then you can run the test command from the [wiki](https://github.com/loosolab/TOBIAS/wiki/BINDetect): 
```$ TOBIAS BINDetect --motifs test_data/motifs.jaspar --signals test_data/Bcell_footprints.bw test_data/Tcell_footprints.bw --genome test_data/genome.fa.gz --peaks test_data/merged_peaks_annotated.bed --peak_header test_data/merged_peaks_annotated_header.txt --outdir BINDetect_output --cond_names Bcell Tcell --cores 8```

That would be really helpful, thank you!

**Comment by adamxmiranda** [comment]
So, Downloading the sample data doesn't seem to be an option for the build of TOBIAS that I have. Do I need to download a different version perhaps?
```
TOBIAS DownloadData --bucket data-tobias-2020
TOBIAS: error: argument : invalid choice: 'DownloadData' (choose from 'ATACorrect', 'ScoreBigwig', 'FootprintScores', 'BINDetect', 'TFBScan', 'FormatMotifs', 'ScoreBed', 'PlotAggregate', 'PlotHeatmap', 'PlotChanges', 'PlotTracks', 'MergePDF', 'MaxPos', 'SubsampleBam', 'CreateNetwork', 'Log2Table')
```

**Comment by adamxmiranda** [comment]
I am running 0.9.0

**Comment by msbentsen** [comment]
Yes "DownloadData" was added in `0.11.0`, so please update to the newest version to use this. Conda is sometimes fetching the wrong version, but I found that `pip install tobias` generally works. 

Maybe this update will also solve the issue, as there were a few bugfixes since 0.9.

**Comment by adamxmiranda** [comment]
Updating to the newer version seems to have worked to solve my problem. Didn't realize mine was so old since I used conda. 

thank you so much
